### ISSUE TITLE
Suggestions/recommendations for DL papers

### ISSUE THREAD

**Issue by act65** [garibaldu/comp421#10]
* [alex net](https://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf)
* [batch norm](https://arxiv.org/abs/1502.03167)
* ?

maybe
* [visualising cnns](https://www.cs.nyu.edu/~fergus/papers/zeilerECCV2014.pdf)
* or [easily fooled](https://arxiv.org/abs/1412.1897)

**Comment by garibaldu** [comment]
Maybe we pick specific example applications from http://machinelearningmastery.com/inspirational-applications-deep-learning/ and the presentation is about the science underneath?

**Comment by garibaldu** [comment]
Word2Vec?  (e.g. could start from summary at http://deeplearning4j.org/word2vec)

**Comment by act65** [comment]
Before FoD I asked Harry and Kelsey what they would prefer out of; new/applied and shiny, old and foundational, theoretical and mathsy. And they went for the latter two. (although maybe their interests dont represent the majority of the class?)

I had been thinking of Resnet for vanishing gradients problem and batch norm for internal covariance shift. But I am a little over these... something new would be nice.

Maybe we should just aim for a balance...? New, old, theory?

Those papers on machine learning mastery look good as well (maybe we could find person/people to just pick one they are interested in?). 

And actually, if you think it's a good idea, I am keen to do the DRAW paper. 

I had the vague hope we could convince Paul (or maybe I could do that???) to introduce us to rnns with some music generated in [Magenta](https://github.com/tensorflow/magenta). but i am unsure about how feasible his would be.

And actually do we even need papers? I mean Andrej Karpathy's blog is just great for [rnns](http://karpathy.github.io/2015/05/21/rnn-effectiveness/) and [RL](http://karpathy.github.io/2016/05/31/rl/)
